jenkins/bootstrap_test.py

```diff
@@ -1416,10 +1426,16 @@ def CheckBootstrapYaml(self, path, check, suffix='suffix', use_json=False):
             self.assertTrue(job_name)
             self.assertEquals(job_name, real_job.get('job-name'))
 
+    def GetRealBootstrapJob(self, job):
+        key = os.path.splitext(job.strip())[0][3:]
```

Comment:
`job_path` is going to be `path/to/jobs/FOO.sh` rather than `bootstrap-foo.yaml`, and jobs are gathered from `config.json` and iter all files from `jobs/` dir. I can list out all the existing yaml files and load them, if a job is not cached yet?